index.bs

```diff
@@ -1040,6 +1185,111 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
       {{Screen}}.
     </p>
   </section>
+
+  <section>
+    <h3 id='video-display-capabilities'>Video Display Capabilities</h3>
+
+    <section>
+      <h4 id='video-display-configuration'>VideoDisplayConfiguration</h4>
+
+      <pre class='idl'>
+        dictionary VideoDisplayConfiguration {
+          required unsigned long width;
+          required unsigned long height;
+          required boolean hasHdrCapabilities;
+        };
+      </pre>
+
+      <p>
+        The <dfn for='VideoDisplayConfiguration' dict-member>width</dfn> member
+        represents the width of the display in physical pixels, not CSS pixels.
+      </p>
+
+      <p>
+        The <dfn for='VideoDisplayConfiguration' dict-member>height</dfn> member
+        represents the height of the display in physical pixels, not CSS pixels.
+      </p>
+
+      <p>
+        The <dfn for='VideoDisplayConfiguration' dict-member>hasHdrCapabilities</dfn>
+        describes the screen's capability to display HDR.
+      </p>
+    </section>
+
+    <section>
+      <h4 id='video-display-info'>VideoDisplayInfo</h4>
+
+      <pre class='idl'>
+        dictionary VideoDisplayInfo {
+          required boolean supported;
+        };
+      </pre>
+
+      <p>
+        A {{VideoDisplayInfo}} has associated <dfn
+        for='VideoDisplayInfo'>supported</dfn> fields which is a boolean.
+      </p>
+
+    <section>
+      <h4 id='Video'>Video</h4>
+
+      <pre class='idl'>
+        [Exposed=(Window)]
```

Comment:
all of this gets re-worked with the comment above to replace method w/ attributes

Reply:
See above comment as we won't be re-working this to keep it as a query